staff/christian-haag/pre-course/calculadora-2/main.js

```diff
@@ -0,0 +1,94 @@
+//Calculadora-2
+//Funcionalidad:
+//Entrada de valores tanto por onClick como por Keypress.
+//Extra: panel informativo que se activa introduciendo 8-7-2019 y Enter. Para que todo vuelva a su sitio, AC o tecla Supr. 
+//------!important---------Para que showInfo() y hideInfo() se activen, el resultanto siempre tiene que ser -2020.
+
+
+function numOp(val) {
+	let doc = document.getElementById('screen').value;
+	 if(doc === '0'){
+		 document.getElementById('screen').value = ''
```

Comment:
ADD semicolon in all lines where is missing

Reply:
Fixed!